public-api.yml

```diff
@@ -780,6 +812,8 @@ components:
           description: List of string codes e.g. BRKF, EARLY, LATE etc or their numeric identifiers
           items:
             type: string
+        metadata:
+          type: array
```

Comment:
this should be an object https://swagger.io/docs/specification/data-models/dictionaries/

Reply:
updated